api/src/main/resources/liquibase.xml

```diff
@@ -8,5 +8,27 @@
         See http://www.liquibase.org/manual/home#available_database_refactorings
         for a list of supported elements and attributes
     -->
+
+    <changeSet id="2018Jun28-1" author="Chew Chia Shao Yuan">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM
+                patient_identifier_type WHERE uuid ='a5d38e09-efcb-4d91-a526-50ce1ba5011a';
+            </sqlCheck>
+        </preConditions>
+        <comment></comment>
+        <insert tableName="patient_identifier_type">
+            <column name="name" value="OpenEMPI ID"/>
```

Comment:
Should say MPI not OpenEMPI because not everybody will use OpenEMPI

Reply:
This is specific to the OpenEMPI Demo, it is configurable, e.g. can use ECID or anything else

Reply:
And that shouldn't be the case, there should be no default values or any logic that is specific to an MPI implementation, everything needs to be agnostic of the MPI implementation.